src/translator/service.cpp

```diff
@@ -202,12 +283,14 @@ void AsyncService::pivot(std::shared_ptr<TranslationModel> first, std::shared_pt
 void AsyncService::translate(std::shared_ptr<TranslationModel> translationModel, std::string &&source,
                              CallbackType callback, const ResponseOptions &responseOptions) {
   // Producer thread, a call to this function adds new work items. If batches are available, notifies workers waiting.
+  // Tagging
+  if (!terminologyMap_.empty()) source = ReplaceTerminology(std::move(source), terminologyMap_);
```

Comment:
`std::string ReplaceTerminology(std::string const &str, TerminologyMap const &terminology);`. I think I made the `std::move` pointless when I changed the `ReplaceTerminilogy` implementation.